src/agent/src/Toolbox/AgentProcessor.php

```diff
@@ -47,7 +47,7 @@ public function __construct(
     public function processInput(Input $input): void
     {
         if (!$input->model->supports(Capability::TOOL_CALLING)) {
```

Comment:
This exception basically says: "If you are providing tools and the model you are using is not capable of using tools, we should inform the developer."

To me your change looks not correct